sphinx_gallery/notebook.py

```diff
@@ -142,36 +145,65 @@ def add_code_cell(work_notebook, code):
     work_notebook["cells"].append(code_cell)
 
 
-def add_markdown_cell(work_notebook, text):
+def add_markdown_cell(work_notebook, text, pandoc=False):
     """Add a markdown cell to the notebook
 
     Parameters
     ----------
     code : str
         Cell content
     """
-    markdown_cell = {
-        "cell_type": "markdown",
-        "metadata": {},
-        "source": [rst2md(text)]
-    }
+    if pandoc:
+        markdown_cell = {
+            "cell_type": "markdown",
+            "metadata": {},
+            "source": [text]
+        }
+    else:
+        markdown_cell = {
+            "cell_type": "markdown",
+            "metadata": {},
+            "source": [rst2md(text)]
+        }
     work_notebook["cells"].append(markdown_cell)
 
 
-def fill_notebook(work_notebook, script_blocks):
+def fill_notebook(work_notebook, script_blocks, gallery_conf):
     """Writes the Jupyter notebook cells
 
+    If available, uses pypandoc to convert rst to markdown.
+
     Parameters
     ----------
     script_blocks : list
         Each list element should be a tuple of (label, content, lineno).
     """
+    pandoc = False
+    pandoc_kwargs = {}
+    if gallery_conf["pypandoc"] or isinstance(gallery_conf["pypandoc"], dict):
+        try:
+            import pypandoc  # noqa
+        except ImportError:
```

Comment:
Also it appears as though `pypandoc` can raise an `OSError` if `pandoc` is not found, so this should be at least:
```
except (ImportError, OSError) as exc:
```
And then below it might be nice to do:
```
"files:\n%s" % (exc,))
```